docs/getting_started/tutorial.rst

```diff
@@ -2,66 +2,128 @@ Your first Wagtail site
+   If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
 
-1. Install Wagtail and its dependencies:
+Install and run Wagtail
+-----------------------
 
-   .. code-block:: console
+Install dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Wagtail supports Python 3.4, 3.5, 3.6, and 3.7.
+
+To check whether you have an appropriate version of Python 3:
+
+.. code-block:: console
+
+   $ python3 --version
+
+If this does not return a version number or returns a version lower than 3.4, you will need to `install Python 3 <https://www.python.org/downloads/>`_.
+
+.. important::
+   Before installing Wagtail, it is necessary to install the **libjpeg** and **zlib** libraries, which provide support for working with JPEG, PNG and GIF images (via the Python **Pillow** library).
+   The way to do this varies by platform—see Pillow's
+   `platform-specific installation instructions <http://pillow.readthedocs.org/en/latest/installation.html#external-libraries>`_.
+
+
+Create and activate a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We recommend using a virtual environment, which provides an isolated Python environment.
+This tutorial uses `venv <https://docs.python.org/3/tutorial/venv.html>`_, which is packaged with Python 3.
+
+**On Windows** (cmd.exe):
+
+    .. code-block:: bat
+
+       $ python3 -m venv mysite\env
+       $ mysite\env\Scripts\activate.bat
 
-       $ pip install wagtail
+**On Unix or MacOS** (bash):
 
-2. Start your site:
+    .. code-block:: console
 
-   .. code-block:: console
+       $ python3 -m venv mysite/env
+       $ source mysite/env/bin/activate
 
-       $ wagtail start mysite
-       $ cd mysite
+**For other shells** see the `venv documentation <https://docs.python.org/3/library/venv.html>`_.
 
-   Wagtail provides a ``start`` command similar to
-   ``django-admin.py startproject``. Running ``wagtail start mysite`` in
-   your project will generate a new ``mysite`` folder with a few
-   Wagtail-specific extras, including the required project settings, a
-   "home" app with a blank ``HomePage`` model and basic templates and a sample
-   "search" app.
+.. note::
+
+   If you're using version control (e.g. git), ``mysite`` will be the directory for your project.
+   The ``env`` directory inside of it should be excluded from any version control.
+
+Install Wagtail
+~~~~~~~~~~~~~~~
+
+Use pip, which is packaged with Python, to install Wagtail and its dependencies:
 
-3. Install project dependencies:
+.. code-block:: console
+
+   $ pip install wagtail
+
+Generate your site
+~~~~~~~~~~~~~~~~~~
 
-   .. code-block:: console
+Wagtail provides a ``start`` command similar to ``django-admin startproject``.
+Running ``wagtail start mysite`` in your project will generate a new ``mysite`` folder with a few Wagtail-specific extras, including
+the required project settings,
+a "home" app with a blank ``HomePage`` model
+and basic templates
```

Comment:
Missing comma at the end of this phrase? Feels like it should read as

> ...including the required project settings, a "home" app with a blank `HomePage` model and basic templates, and a sample "search" app.

Reply:
Changed. Thanks for pointing that out :)  (I had noticed that something was confusing when I split this paragraph into multiple lines like this, and had forgotten to go back to it.)